### DISCUSSION TITLE
Community question: What improvements are desired?

### DISCUSSION THREAD

**Discussion by rzeldent** [rzeldent/esp32-smartdisplay#49] [Polls]
This library has been around for some time. I see a lot of clones and occasionally a project with this library.
But what is holding you back? Are there other annoyances?

Let me know in the poll below...
![_c0fb91ea-2ffd-4a25-9813-58f285fc3dc4](https://github.com/rzeldent/esp32-smartdisplay/assets/7620231/cb17af1d-c18a-4562-9194-59614d83446c)

**Comment by mhaberler** [comment]
thanks for that great project which really accelerates getting started!

- an [EEZ Studio](https://github.com/eez-open/studiohttps://github.com/eez-open/studio) demo as alternatie to [Squareline](https://squareline.io/)
- LVGL 9 support
- a test case for SDcard I/O which is frequently an issue with boards haing shared SPI between display and SD card - ideally also with [SdFat](https://github.com/greiman/SdFat) 

Questions: 

- are you entertaining [PR's for non-Sunton boards](https://github.com/rzeldent/platformio-espressif32-sunton)?
- is an ESP-IDF only version something you'd consider?

**Comment by gavzzorz** [comment]
Add support for Waveshare ESP32-S3-Touch-LCD-4

i see you have support for the 7" version that is similar so should be a quick add.
https://www.waveshare.com/wiki/ESP32-S3-Touch-LCD-4

i tried adding a board conf for it but i might have missed something as i just have a black screen where previously i got a display with the same project with board ESP32-2432S032C.

Thanks for a great project that really helps with these ESP32 Displays :)
